src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -173,6 +173,30 @@ def function_converter(self) -> Mapping[str, fields.MetricsFunction]:
                     ),
                     default_result_type="integer",
                 ),
+                fields.MetricsFunction(
+                    "division_if",
+                    required_args=[
+                        fields.MetricArg(
+                            # needs to be named column, otherwise the query builder won't be able to determine the correct target table
+                            "column",
+                            allow_custom_measurements=False,
+                        ),
+                        fields.MetricArg(
+                            "column1",
```

Comment:
Can we give the `column1` a more descriptive name? Maybe like `divisorColumn` to match its role in the division.